tools/toolchains/iar.py

```diff
@@ -89,6 +89,10 @@ def __init__(self, target, notify=None, macros=None,
         self.cppc = [main_cc]
         self.cc += self.flags["common"] + c_flags_cmd + self.flags["c"]
         self.cppc += self.flags["common"] + c_flags_cmd + cxx_flags_cmd + self.flags["cxx"]
+        if target.name in ["REALTEK_RTL8195AM"]:
+            self.cc += ["--enum_is_int"]
+            self.cppc += ["--enum_is_int"]
```

Comment:
@0xc0170 That does not skip the execution of `ielftool` like requested. What special script do you have to do elf2bin?